example-java-kotlin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Apr 12 18:39:10 SGT 2017
+#Wed Oct 25 16:11:14 BST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.2.1-all.zip
```

Comment:
See https://github.com/btkelly/gnag/pull/166/commits/198ce356c8ca4d28acbb2a33e1a22e1a1004ef0c

Reply:
I'm not sure that this matters any more since we're so explicit about handling paths...